models/mailchimp/mailchimp_emails_denormalized.sql

```diff
@@ -0,0 +1,53 @@
+with email_actions as (
+    select *
+    from {{env.schema}}.mailchimp_email_actions
+),
+
+sends as (
+    select campaign_id, email_id, sent_date
+    from {{env.schema}}.mailchimp_sent_to
+),
+
+bounces as
+(
+    select campaign_id, email_id, min(action_date) as bounced_date
+    from email_actions
+    where action = 'bounce'
+    group by campaign_id, email_id
+),
+
+opens as (
+    -- look at the first open date
```

Comment:
We shouldn't be looking at the first open date here. It's completely legitimate to open an email multiple times, that's not something we want to scrub out. If we want to analyze distinct opens, that query should happen at the analysis layer.